docs/cody/core-concepts/index.mdx

```diff
@@ -8,9 +8,9 @@
 
 <QuickLink title="Keyword Search" icon='presets' href="/cody/core-concepts/keyword-search" description="Learn how Cody makes use to Keyword Search to gather context." />
 
-<QuickLink title="Cody Gateway" icon='theming' href="/cody/core-concepts/cody-gateway" description="Learn how Cody Gateway powers the default Sourcegraph provider for completions to facilitate Cody features." />
+<QuickLink title="Sourcegraph Model Provider" icon='theming' href="/model-provider" description="Learn how the default Sourcegraph model provider facilitates Cody features." />
```

Comment:
title case here and in other places not - I think we should decide on one of the two. I'd be fine with Title Case and making it a product-y name but your call

Reply:
I don't really understand the casing - I think we Title Case all page titles and headers, which is why there's a difference right now

But I think since I'm pushing it to a top-level concept anyway maybe just Title Case the thing universally is good